General structure of government

Defiant-Sheep opened the pull request:
This time I have only a description of the basic structure of the federal government. I believe that what I have written is enough for this chapter.

- This structure includes a bicameral legislature. This is relatively common among legislatures of federations. The upper house is intended to protect the independence that the states retain. It is a common institutional design among most federations.
- The cabinet is responsible to both the President and the lower house. The input of the lower house tries to ensure that the cabinet is working for most Europeans. However, the President can remove ministers aligned with the legislature if the latter is acting against the interests of the nation.
- I am giving the ability to impeach presidents to a body selected by sortition. I am hoping that this will make it easier to remove corrupt presidents compared to giving a supermajority of the legislature this power.

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

The basic structure of government

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defined the election of the cabinet

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Specified the basic legislative procedure.

Defiant-Sheep committed:
Senate Nominations and Election

I believe that this method of nominating candidates to the senate will help to avoid the cronyism that many upper houses suffer from

Defiant-Sheep committed:
Timing of elections

Defiant-Sheep committed:
European Parliament Elections

Made a more sensical system than the one I had before

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

I read the wiki properly today, and finally realised that you wanted the executive branch to have significant independence from the legislature. I am rewriting all of this into a semi-presidential system.

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

Defiant-Sheep committed:
Legislative Process

Defiant-Sheep committed:
Removed misplaced description of legislative process

Defiant-Sheep committed:
Merge pull request #1 from Defiant-Sheep/Semi-Presidential

I have realised that the main branch serves no clear purpose

Defiant-Sheep committed:
Presidential Elections

I am suggesting that nominations be done by member states to increase the separation of powers. Nomination by petition would lead to hundreds of candidates running, while nomination by the Continental Congress would tie them too closely with their congressional allies. I think member states should nominate multiple candidates to increase diversity of choice.

The first round must happen to allow voters to know who the frontrunners are and to learn about them. The first two parts of the process produce candidates that are trusted outside of their member state, so that it cannot be skewed by the large states. The second two rounds ensure that there are genuine opposition candidates running in case no candidate is approved of by a majority. 

The second round is a Condorcet system. This means that if one candidate is preferred to every other candidate, they will win, even if very few people saw them as the best option. This is less certain in Instant Runoff Voting.

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TABLE_OF_CONTENTS.md

Defiant-Sheep committed:
Impeachment

I am adding an institution who can remove the president. We can see in from Trump's second impeachment that the legislature will contain too many loyalists to the president when there is a valid reason to impeach them.

Defiant-Sheep committed:
Practicality concerns

Defiant-Sheep committed:
Created a process to impeach the president

Defiant-Sheep committed:
I figured out how to make a proper list

Defiant-Sheep committed:
Ensured that the Federal Jury contains no lifetime members

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

Defiant-Sheep committed:
Update TABLE_OF_CONTENTS.md

Defiant-Sheep committed:
Presidential succession and no confidence required for vice presidents

I forgot to add a system for the absence of a president earlier. I also realised that requiring parliamentary approval for their appointment would damage separation of powers in this case.

Defiant-Sheep committed:
Missed stuff in last commit

Defiant-Sheep committed:
Changed "Continental Congress" to simply "Congress"

Defiant-Sheep committed:
Appointment of ambassadors and ratification of treaties

Defiant-Sheep committed:
Mentioned the lower house's powers over the cabinet in the structure of government

Defiant-Sheep committed:
Renamed "Congress" to the "Consilium"

The name was suggested by Mathew Irone. I like it the best out of all the names either of us have thought of.

Defiant-Sheep committed:
Renamed "Congress" to the "Consilium"

Defiant-Sheep committed:
Moved the Structure of Government to its own branch

Staphylococcus asked me to separate this so that the pull requests could be done in manageable portions.

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

After looking at this again, I realised that "Enabling Acts" is a poor term for these kinds of laws.

Defiant-Sheep committed:
Constituency seat allocations

I have no recollection of why I decided to increate the population required for a country to gain more seats in the European Parliament. This is fixed now.

Defiant-Sheep committed:
Defined the Single Transferrable Vote

Defiant-Sheep committed:
Description of referendum

I think that referendums should require a minimum threshold of voter participation to ensure that the people actually wanted them to pass.

Mathew-Iron reviewed (changes requested):
it's great but the announcement of the existence of the president and what he means should be introduced before mentioning said president and the jury chosen by sortition risks creating a problem, juror chosen at random, it lacks clarity and therefore risks being misinterpreted

Defiant-Sheep committed:
Moved president article

I was informed that the president should be described before the cabinet

Defiant-Sheep committed:
Definition for Sortition

I codified a simple definition for sortition.

I cannot think of any reason I might of had for giving all but 2 member states the same number of Federal Jurors, so I changed the math.

Defiant-Sheep commented:
I am very happy that you think my structure is generally good. Do you feel that I have clarified the meaning of sortition this time?

Defiant-Sheep committed:
Merge branch 'main' into General-Structure-of-Government

Defiant-Sheep committed:
Removed inadvertant merge

I accidentaly added out of scope content to this branch

Defiant-Sheep committed:
Removed inadvertant merge

I accidentaly added out of scope content to this branch

Defiant-Sheep committed:
Removed inadvertant merge

I accidentaly added out of scope content to this branch

Defiant-Sheep commented:
@Mathew-Iron I am thinking about if replacing the chancellor in my model with a committee might be a good idea? A committee might force the president to consider the opinions of the entire governing coalition rather than just his own and the chancellor's, but a single chancellor could provide greater stability. What are your thoughts:?

Staphylococcus commented:
@Mathew-Iron Which points prevent us from merging this pull request? Let's keep things simple, resolve those, merge, and work forward in a different branch.
